Documentation/contributors/design-proposals/subscription-status.md

```diff
@@ -0,0 +1,188 @@
+# Improved Subscription Status
```

Comment:
Does it make sense for us to follow the main bullets defined in the [kube proposal template](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/Design_Proposal_TEMPLATE.md)?